modules/gapi/include/opencv2/gapi/gcall.hpp

```diff
@@ -56,11 +56,14 @@ class GAPI_EXPORTS GCall final
     Priv& priv();
     const Priv& priv() const;
 
-protected:
-    std::shared_ptr<Priv> m_priv;
+    GKernel& kernel();
+    cv::util::any& params();
```

Comment:
Are params modifiable?

Reply:
Comments are required on these fields

Reply:
Yes `params` are modifiable, because `GInferOuput::at` expands `output_names` which stored in `params`

Reply:
Comment

Reply:
Added